doc/release_notes/index.rst

```diff
@@ -16,3 +16,4 @@ Release notes
 .. include:: v0.9.4.rst
 .. include:: v0.9.3.rst
 .. include:: v0.9.2.rst
+.. include:: v2.0.0.rst
```

Comment:
Forgot to add this, obviously... shouldn't it be added at the top?

Reply:
oops - yep -- I'll correct that a bit later - about to go out.